packages/camera/lib/src/common/camera_channel.dart

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart';
+
+typedef CameraCallback = void Function(dynamic result);
+
+class CameraChannel {
+  static final Map<int, dynamic> callbacks = <int, CameraCallback>{};
```

Comment:
This will be used for callback such as `Camera.ErrorCallback`
https://developer.android.com/reference/android/hardware/Camera.ErrorCallback